image-meta-tag/meta.yaml

```diff
@@ -0,0 +1,28 @@
+package:
+    name: image-meta-tag
+    version: "0.4"
+
+source:
+    git_url: https://github.com/SciTools-incubator/image-meta-tag
+    git_tag: 0.4.1
+
+build:
+    number: 0
```

Comment:
Ah, quite right. I don't think skipping them will make any difference though; the failure occurs much before the build stage has even started.